src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class Main {
+    private static FigureSupplier figureSupplier = new FigureSupplier();
+    private static Figure[] figures = new Figure[6];
+
```

Comment:
https://mate-academy.github.io/jv-program-common-mistakes/java-core/abstract-class-interface/oop-advanced#all-magic-numbers-in-your-code-should-be-constants

Reply:
final ??

Reply:
Thank you